sw/tools/attitude_viz.py

```diff
@@ -338,7 +338,7 @@ def run():
     VEHICLE_QUATS = [["AHRS_REF_QUAT", 6, "Estimate", True], ["AHRS_REF_QUAT", 2, "Reference", True]]
     BAR_VALUES = [["ROTORCRAFT_RADIO_CONTROL", 5, "Throttle (%%) %i", 0, 100, 100]]
     window_title = "Attitude_Viz"
-    rotate_theta = -90
```

Comment:
I have not used the visualizer recently on master, but it used to be rotated by 90 degrees wrt. the quadshot attitude... Not sure what help you mean